administration/src/bp-modules/self-service/hooks/__tests__/useCardGeneratorSelfService.test.tsx

```diff
@@ -40,6 +30,24 @@ jest.mock('../../../../cards/PdfFactory', () => ({
 
 jest.mock('../../../../util/downloadDataUri')
 
+const wrapper = ({ children, initialRoutes }: { children: ReactNode; initialRoutes?: string[] }) => (
+  <MemoryRouter initialEntries={initialRoutes}>
+    <AppToasterProvider>
+      <MockedProvider mocks={mocks} addTypename={false}>
+        <ProjectConfigProvider projectConfig={koblenzConfig}>{children}</ProjectConfigProvider>
+      </MockedProvider>
+    </AppToasterProvider>
+  </MemoryRouter>
+)
+
+const withCustomWrapper =
+  (initialRoute: string) =>
+  ({ children }: { children: ReactNode }) =>
+    wrapper({
+      children,
+      initialRoutes: [initialRoute],
+    })
```

Comment:
:upside_down_face: I guess we should make this reusable at some point to a shared testing file.

Reply:
You could also just straight move it to a helper? :D